WIP/Source/Veazy/src/test/java/vn/edu/fu/veazy/core/service/LessonServiceTest.java

```diff
@@ -0,0 +1,377 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+package vn.edu.fu.veazy.core.service;
+import java.util.List;
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.test.context.ContextConfiguration;
+import org.springframework.test.context.junit4.SpringJUnit4ClassRunner;
+import vn.edu.fu.veazy.core.form.CreateLessonForm;
+import vn.edu.fu.veazy.core.response.CreateLessonResponse;
+import vn.edu.fu.veazy.core.form.RegisterForm;
+import vn.edu.fu.veazy.core.form.UpdateLessonForm;
+import vn.edu.fu.veazy.core.model.UserModel;
+import vn.edu.fu.veazy.core.response.BriefLessonResponse;
+import vn.edu.fu.veazy.core.response.GetLessonResponse;
+import vn.edu.fu.veazy.core.response.GetLessonVersionResponse;
+
+/**
+ *
+ * @author Hoang Linh
+ */
+@RunWith(SpringJUnit4ClassRunner.class)
+@ContextConfiguration(locations = {"classpath:testContext.xml"})
+public class LessonServiceTest {
+    
+	private UserModel creatorModel;
+    private UserModel learnerModel;
+    @Autowired
+    private LessonService lessonService;
+    @Autowired
+    private UserService userService;
+
+    private Integer creatorId;
+    private Integer learnerId;
+    private CreateLessonForm createForm;
+    private UpdateLessonForm updateForm;
+    private Integer courseId;
+    
+	public UserModel setUpUser(String userName) throws Exception {
+        RegisterForm form = new RegisterForm();
+        form.setUsername(userName);
+        form.setEmail(userName);
+        form.setPassword(userName);
+        userService.saveUser(form);
+        UserModel user = userService.findUserByUsername(userName);
+        return user;
+    }
+
+    public CreateLessonResponse setUpLesson(Integer creatorId, Integer courseId) throws Exception {
+        CreateLessonForm lessonForm = new CreateLessonForm();
+        lessonForm.setCourseId(courseId);
+        CreateLessonResponse createLesson = lessonService.createLesson(creatorId, lessonForm);
```

Comment:
cái này là tôi dùng hàm save user = RegisterForm nên chỉ có các trường như
vậy nhé


On Fri, Aug 19, 2016 at 5:25 PM, Nguyễn Ngọc Minh <notifications@github.com>
wrote:

> In WIP/Source/Veazy/src/test/java/vn/edu/fu/veazy/core/
> service/LessonServiceTest.java
> <https://github.com/sangnvus/201605JS01/pull/25#discussion_r75459492>:
>
> > +    private Integer courseId;
> > +
> > +	public UserModel setUpUser(String userName) throws Exception {
> > +        RegisterForm form = new RegisterForm();
> > +        form.setUsername(userName);
> > +        form.setEmail(userName);
> > +        form.setPassword(userName);
> > +        userService.saveUser(form);
> > +        UserModel user = userService.findUserByUsername(userName);
> > +        return user;
> > +    }
> > +
> > +    public CreateLessonResponse setUpLesson(Integer creatorId, Integer courseId) throws Exception {
> > +        CreateLessonForm lessonForm = new CreateLessonForm();
> > +        lessonForm.setCourseId(courseId);
> > +        CreateLessonResponse createLesson = lessonService.createLesson(creatorId, lessonForm);
>
> tương tự như trên, thiếu các trường not null
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/sangnvus/201605JS01/pull/25/files/442cb029a2ed3ce65a0c4e6062ce71d64bbc74d1#r75459492>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ASfcY-QurpCOcra95ECYJhaI7dK6hvBUks5qhYSWgaJpZM4JnxSY>
> .
>